### PR TITLE
Fix ES Linting issues for leaderboard.js

### PR DESCRIPTION
**Pull request by tra38**
There were a few problems according to ESLint,
and Docker would not allow me to continue work on Issue #239,
until they were corrected. Here is a list of those errors:

node_1     |
node_1     | /node/app/assets/v2/js/pages/leaderboard.js
node_1     |    4:57  error  Unexpected space before function parentheses     space-before-function-paren
node_1     |    4:60  error  Missing space before opening brace               space-before-blocks
node_1     |    7:5   error  Expected blank line after variable declarations  newline-after-var
node_1     |   11:40  error  There must be a space inside this paren          space-in-parens
node_1     |   11:42  error  There must be a space inside this paren          space-in-parens
node_1     |   11:43  error  Missing space before opening brace               space-before-blocks
node_1     |   16:33  error  Missing space before opening brace               space-before-blocks
node_1     |
node_1     | ✖ 7 problems (7 errors, 0 warnings)
node_1     |   7 errors, 0 warnings potentially fixable with the `--fix` option.
node_1     |

##### Description

Fixes ES Linting issue

##### Checklist
- [X] linter status: 100% pass
- [X] changes don't break existing behavior
- [X] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

leaderboard.js

### CODE REVIEW

File: app/assets/v2/js/pages/leaderboard.js [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+  $( '.leaderboard_entry .progress-bar' ).each( function() {
     const max = parseInt( $( this ).attr( 'aria-valuemax' ) );
     const now = parseInt( $( this ).attr( 'aria-valuenow' ) );
     const width = ( now * 100 ) / max;
+
     $( this ).css( 'width', `${width}%` )
   })
 
-  $( '.clickable-row' ).click( function(e){
+  $( '.clickable-row' ).click( function( e ) {
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/leaderboard.js]: @mbeacom are we fixed on this format ? 
 It looks odd `( foo )` as compared to `(foo)`

**Comment by michelgotta** [comment]
I also checked this file and all the eslint configs. I think the used style in all JS files could be improved. I think using a common standard (I know, this is hard to define) should be used. 

@thelostone-mc I'm with you, we should remove all those unnecessary spaces.

**Comment by thelostone-mc** [comment]
@michelgotta If you've got the time, care to update the eslint config and shoot out a PR ? ^_^

**Comment by michelgotta** [comment]
I just started to scan through the JS code. I will check what I can do. Do you know who got the lead on the JS?

**Comment by thelostone-mc** [comment]
Ah no lead for JS bit yet, but I was around when the config was set up! I'm happy to help out ^_^ 

But if you need a name, I'd say it's @mbeacom

**Comment by tra38** [comment]
If we are going to update the ESLint config then, do we really need to merge this PR in?

**Comment by thelostone-mc** [comment]
@tra38 sounds good to me ^_^

**Comment by mbeacom** [comment]
@tra38 I'm fine with merging this for the moment. I don't want you blocked until we update the eslint configuration.

**Pull request merged**